pandas/_libs/tslibs/tzconversion.pyx

```diff
@@ -123,6 +122,7 @@ timedelta-like}
     elif nonexistent == 'shift_backward':
         shift_backward = True
     elif PyDelta_Check(nonexistent):
+        from .timedeltas import delta_to_nanoseconds
```

Comment:
does it make sense to move delta_to_nanoseconds even higher eg to base.pyx ?

Reply:
ive considered this and gotten hung up on attribute lookup behavior which isnt doing optimizations that i think it should.  I'd like to punt on this for now until I get that sorted out.